### PR TITLE
Update product-led metric definitions

### PR DESCRIPTION
**Pull request by jasonyavorsky**
This is intended to implement the agreed upon proposals from https://docs.google.com/document/d/1o0dLmdRRI6uWIuAg_8VQw25KnTM1CBDKKR2K91SxpAI/edit?skip_itp2_check=true&pli=1# into the handbook.

Created from https://github.com/sourcegraph/about/pull/4521 after handbook refactor (the bulk of the discussion for this PR is there).

### CODE REVIEW

**Comment by jasonyavorsky** [comment]
@anna-mmikhova @serinadawn @beyang @kelsey-brown @serinadawn @bcreager @christinaforney @rrhyne @dadlerj @ryphil @ebrodymoore @malomarrec just as a heads up I've moved the content from https://github.com/sourcegraph/about/pull/4521 into this PR due to the handbook refactor.

Christina, this is just pending your confirmation that this is correct (in covering for Anna while she is away, and just to make sure I've not missed anything given I'm relatively new). Next step will be a PR to set an actual goal/update OKR, which I'll follow up on with you once we've merged this.

**Comment by nicksnyder** [comment]
This clarity is great, what is blocking this from being merged?

File: content/bizops/user_definitions.md
Comment (jasonyavorsky) [inline content/bizops/user_definitions.md]:
```suggestion
For some of our metrics we want to identify unique users across on-prem and cloud instances. For this purpose, unique users of Sourcegraph are defined as having a common email address across any number of instances. For example, a@b.com on an on-prem instance and a@b.com on Sourcegraph.com would be considered the same user for the purposes of this metric. Note that most measures don't have this constraint, and any that do will include the term "Unique Sourcegraph Users" in the name of the metric.
```

File: content/bizops/user_definitions.md
Comment (jasonyavorsky) [inline content/bizops/user_definitions.md]:
```suggestion
A visitor is a unique person (unique to the instance being measured) who showed up on the site and did anything (or nothing at all, apart from viewing the page.) This is important to measure because each of these users is potentially an active user, but it's also important that we don't count anyone who uses the product as an active user so that we can measure success of our features that are intended to convert them.
```

File: content/bizops/user_definitions.md
Comment (jasonyavorsky) [inline content/bizops/user_definitions.md:49]: @ebrodymoore this brief section is new and is meant to define an optional criteria to avoid double-counting users who have a .com account and an on-prem account, in cases when we're looking at user counts in total, which we're doing in relation to OKRs but also could be useful in other scenarios. What do you think?

**Review by christinaforney — APPROVED** [review]

File: content/bizops/user_definitions.md
Comment (jasonyavorsky) [reply content/bizops/user_definitions.md:49]: @ebrodymoore as this PR is now approved, I'm going to go ahead and merge it. But I would still want to get your feedback on this, we can just make any needed changes in a follow up PR.

File: content/bizops/user_definitions.md
Comment (dadlerj) [inline content/bizops/user_definitions.md:12]: @jyavorska once we eventually have a strong/reliable grasp on what this actually means, we can definitely start sending this count back in pings as well

**Pull request merged**
